qiskit/extensions/standard/iswap.py

```diff
@@ -0,0 +1,122 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+iSWAP gate.
+"""
+
+import numpy as np
+from qiskit.circuit import Gate
+from qiskit.circuit import QuantumCircuit
+from qiskit.circuit import QuantumRegister
+
+
+class iSwapGate(Gate):
+    r"""iSWAP gate.
+
+    A 2-qubit XY interaction that is equivalent to a SWAP up to a diagonal.
+    This is a Clifford and symmetric gate. It's action is to swap two qubit
+    states and phase the |01⟩ and |10⟩ amplitudes by i.
```

Comment:
This should fix that doc error:

```suggestion
    states and phase the :math:`|01\\rangle` and :math:`|10\\rangle` amplitudes by i.
```

Reply:
oh i see let me try this thanks!